steps/PageTitles.ts

```diff
@@ -43,6 +42,7 @@ enum PageTitles {
   Pets = "Pets",
   AntisocialBehaviour = "Antisocial behaviour",
   OtherComments = "Other comments",
+  Repairs = "Repairs",
```

Comment:
Let's move this back to where it was. The order of this enum helps see the order of the pages.